restclient/src/main/java/com/box/l10n/mojito/rest/resttemplate/ResttemplateConfig.java

```diff
@@ -56,6 +57,29 @@ public enum CredentialProvider {
     }
   }
 
+  public String getURIForResource(String resourcePath) {
```

Comment:
Can you please add utility class for this. Let's keep the Config class just to read properties files.

Reply:
Sure, created a RestTemplateUtil class to place the getURIForResource method. Thanks